doc/modules/mod_private.md

```diff
@@ -6,7 +6,7 @@ This module implements [XEP-0049: Private XML Storage](http://xmpp.org/extension
 * `backend` (atom, default: `mnesia`): Storage backend. Currently `mnesia`, `rdbms`, `riak` and `mysql` are supported . `mysql` uses MySQL-specific queries so in some cases it is more efficient than generic `rdbms`.
 
 **CAUTION:**  Riak KV backend doesn't support transactions (rollbacks), so please avoid inserting more
-than one value in one set request, otherwise you may end up with partially save data, backend returns
+than one value in one set request, otherwise you may end up with partially saved data. Backend returns
```

Comment:
**CAUTION:**  Riak KV backend doesn't support transactions (rollbacks), so please avoid inserting more than one value in a single set request, otherwise you may end up with partially saved data. Backend returns the first error.

?